test/hotspot/jtreg/runtime/Thread/TestThreadDumpMonitorContention.java

```diff
@@ -50,6 +50,7 @@ public class TestThreadDumpMonitorContention {
     // getJDKTool() which can fall back to "compile.jdk".
     final static String JSTACK = JDKToolFinder.getTestJDKTool("jstack");
     final static String PID = Long.toString(ProcessHandle.current().pid());
+    final static String DISABLE_STREAMING_OUTPUT = "-J-Djdk.attach.allowStreamingOutput=false";
```

Comment:
Would you mind writing a comment here explaining why this is needed for this test?